build.gradle

```diff
@@ -55,6 +44,10 @@ dependencies {
   testCompile 'org.mockito:mockito-core:2.7.21'
 }
 
+task wrapper(type: Wrapper) {
+  gradleVersion = '3.4.1'
```

Comment:
Maybe this as well? (to `4.9`)

Reply:
hrmm... I think that might trigger some things I'm not ready to clean up :\. Lemme see what happens